src/auth.ts

```diff
@@ -1,93 +1,22 @@
-import * as jose from "jose"
-import { AccessToken, createAccessToken, CreateAccessTokenRequest } from "./api/accessToken"
-import {
-    ApiKeysCreateRequest,
-    ApiKeysQueryRequest,
-    ApiKeyUpdateRequest,
-    createApiKey,
-    deleteApiKey,
-    fetchApiKey,
-    fetchArchivedApiKeys,
-    fetchCurrentApiKeys,
-    updateApiKey,
-    validateApiKey,
-} from "./api/endUserApiKeys"
-import { createMagicLink, CreateMagicLinkRequest, MagicLink } from "./api/magicLink"
-import { migrateUserFromExternalSource, MigrateUserFromExternalSourceRequest } from "./api/migrateUser"
-import {
-    addUserToOrg,
-    AddUserToOrgRequest,
-    allowOrgToSetupSamlConnection,
-    changeUserRoleInOrg,
-    ChangeUserRoleInOrgRequest,
-    createOrg,
-    CreateOrgRequest,
-    deleteOrg,
-    disallowOrgToSetupSamlConnection,
-    fetchOrg,
-    fetchOrgByQuery,
-    OrgQuery,
-    OrgQueryResponse,
-    removeUserFromOrg,
-    RemoveUserFromOrgRequest,
-    updateOrg,
-    UpdateOrgRequest,
-} from "./api/org"
-import {
-    fetchTokenVerificationMetadata,
-    getTokenVerificationMetadataPromise,
-    TokenVerificationMetadata,
-    TokenVerificationMetadataWithPublicKey,
-} from "./api/tokenVerificationMetadata"
-import {
-    clearUserPassword,
-    createUser,
-    CreateUserRequest,
-    deleteUser,
-    disableUser,
-    disableUser2fa,
-    disableUserCanCreateOrgs,
-    enableUser,
-    enableUserCanCreateOrgs,
-    fetchBatchUserMetadata,
-    fetchUserMetadataByQuery,
-    fetchUserMetadataByUserIdWithIdCheck,
-    fetchUsersByQuery,
-    fetchUserSignupQueryParams,
-    fetchUsersInOrg,
-    inviteUserToOrg,
-    InviteUserToOrgRequest,
-    updateUserEmail,
-    UpdateUserEmailRequest,
-    updateUserMetadata,
-    UpdateUserMetadataRequest,
-    updateUserPassword,
-    UpdateUserPasswordRequest,
-    UserSignupQueryParams,
-    UsersInOrgQuery,
-    UsersPagedResponse,
-    UsersQuery,
-} from "./api/user"
-import { ForbiddenException, UnauthorizedException } from "./exceptions"
 import {
-    ApiKeyFull,
-    ApiKeyNew,
-    ApiKeyResultPage,
-    ApiKeyValidation,
-    CreatedOrg,
-    CreatedUser,
+    ForbiddenException,
+    getApis,
     InternalUser,
-    Org,
-    OrgApiKeyValidation,
     OrgIdToOrgMemberInfo,
     OrgMemberInfo,
-    PersonalApiKeyValidation,
+    TokenVerificationMetadata,
     toUser,
+    UnauthorizedException,
     User,
     UserAndOrgMemberInfo,
-    UserMetadata,
-} from "./user"
-import { validateAuthUrl, validateOrgApiKey, validatePersonalApiKey } from "./validators"
+} from "@propelauth/node-apis"
+import * as jose from "jose"
+import { UserClass } from "../.yalc/@propelauth/node-apis"
```

Comment:
This needs to come from the published node-apis rather than the yalc-published build